# Adjust joint limits according to the specifications of the UR robot
joint_limits:
  elbow_joint:
    has_velocity_limits: true
    max_velocity: 3.1415926535897931
    has_acceleration_limits: True
    max_acceleration: 12.8
  shoulder_lift_joint:
    has_velocity_limits: true
    max_velocity: 2.0943951023931953
    has_acceleration_limits: true
    max_acceleration: 3.2
  shoulder_pan_joint:
    has_velocity_limits: true
    max_velocity: 2.0943951023931953
    has_acceleration_limits: true
    max_acceleration: 3.2
  wrist_1_joint:
    has_velocity_limits: true
    max_velocity: 3.1415926535897931
    has_acceleration_limits: true
    max_acceleration: 3.2
  wrist_2_joint:
    has_velocity_limits: true
    max_velocity: 3.1415926535897931
    has_acceleration_limits: true
    max_acceleration: 3.2
  wrist_3_joint:
    has_velocity_limits: true
    max_velocity: 3.1415926535897931
    has_acceleration_limits: true
    max_acceleration: 3.2